app/models/arm.rb

```diff
@@ -44,7 +44,8 @@ class Arm < ActiveRecord::Base
   after_save :update_liv_subject_counts
 
   validates :name, presence: true
-  validates_format_of :name, with: /\A([A-Za-z0-9][A-Za-z0-9]*([ ][A-Za-z0-9])?)*\z/
+  validates :name, format: { with: /\A([A-Za-z0-9][A-Za-z0-9]*([ ][A-Za-z0-9])?)*\z/, 
+                   message: "can not contain any of the following characters: [ ] * / \\ ? : " }
```

Comment:
I think you can YMLize the error message. See ActiveRecord I18n documentation
http://guides.rubyonrails.org/i18n.html#translations-for-active-record-models